### PR TITLE
Feat: form layout input

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#425
 Closes SAP/fundamental-styles#426

## Description
Adding layout grid to fix alignments

### Before:
No grid available

### After:
Screen Shot 2019-11-15 at 1 59 30 PM
Screen Shot 2019-11-15 at 1 59 23 PM

### CODE REVIEW

**Comment by jbadan** [comment]
Is this a fiori3 component?

**Comment by stefanoScalzo** [comment]
> Is this a fiori3 component?

yes

**Comment by droshev** [comment]
@jbadan the changes are reflecting Fiori 3 specs

File: src/form-layout.scss [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+.fd-form-horizontal
+*/
+
+$block: #{$fd-namespace}-form-layout;
+
+.#{$block} {
+  @include fd-reset();
+
+  background-color: transparent;
+  background-color: var(--sapGroup_ContentBackground);
```
Comment (InnaAtanasova) [inline src/form-layout.scss]: In the specs, the background is `transparent`. Why are you setting it first to `transparent` and then to `var(--sapGroup_ContentBackground)`?

File: src/form-layout.scss [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  padding: 0.5rem 1rem 1rem;
+  grid-gap: var(0.5rem, 0.5rem);
+  width: 100%;
+
+  &_horizontal--4-7-1 {
```
Comment (InnaAtanasova) [inline src/form-layout.scss]: I think we can remove the 4-7-1 from the class name.
could be just `fd-form-layout--horizontal` which will be 4-7-1 layout, and `fd-form-layout--vertical` which will be 12

@droshev , what do you think?

File: src/form-layout.scss [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      min-width: 0;
+
+      .fd-input {
+        margin-top: 0.25rem;
+        margin-bottom: 0.25rem;
+        height: 2.25rem;
+
+        &--compact {
+          margin-top: 0.1875rem;
+          margin-bottom: 0.1875rem;
```
Comment (InnaAtanasova) [inline src/form-layout.scss]: These values are repeated a lot. Maybe a good idea is to make them variables

File: src/form-layout.scss [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+      .fd-form-label {
+        align-self: center;
+        justify-self: right;
+        padding-left: 0.75rem;
+        margin-right: 0.75rem;
+
+        &--compact {
+          margin-top: 0.1875rem;
+          margin-bottom: 0.1875rem;
```
Comment (InnaAtanasova) [inline src/form-layout.scss]: These values are repeated a lot. Maybe a good idea is to make them variables

**Comment by InnaAtanasova** [comment]
Screen Shot 2019-11-21 at 4 34 37 PM
Looks like when you have two inputs inline, the padding inside the inout is missing.

The second issue from the screenshot is the alignment of the label text. The `Password:*` is not really in the center

**Comment by InnaAtanasova** [comment]
There's a `</br>` tag at the bottom of the doc page. Also, I think we need some description for the Grif Layouts (some short explanation under the 4-7-1 and 12 layouts)

**Comment by stefanoScalzo** [comment]
Will be implemented in new PR as inline forms have changed completely

**Pull request closed without merging**